### ISSUE TITLE
Wrong data file name in path when downloading gapminder data: working with data frames

### ISSUE THREAD

**Issue by budell** [swcarpentry/r-novice-gapminder#421]
---
In lesson 5: Working with Data Frames, there is a miscellaneous tip on how to download the gapminder data directly from a web address, followed by this code with throws a 404 not found error: 
gapminder <- read.csv("https://swcarpentry.github.io/r-novice-gapminder/data/gapminder-FiveYearData.csv")

The problem is that the data file has been renamed to gapminder_data.csv.  The issue can be fixed by changing the previous line to: 

gapminder <- read.csv("https://swcarpentry.github.io/r-novice-gapminder/data/gapminder_data.csv")

---

**Comment by jcoliver** [comment]
Thanks, @budell . This is part of a recent change that is still being reconciled (#417 , #418 ).

**Comment by naupaka** [comment]
Looks like a lot of places need fixed, ASAP:

```sh
$ grep -n "FiveYearData.csv" */*.Rmd
_episodes_rmd/02-project-intro.Rmd:148:> Download the gapminder data from [here](https://raw.githubusercontent.com/resbaz/r-novice-gapminder-files/master/data/gapminder-FiveYearData.csv).
_episodes_rmd/02-project-intro.Rmd:151:> 2. Make sure it's saved under the name `gapminder-FiveYearData.csv`
_episodes_rmd/02-project-intro.Rmd:170:> > ls -lh data/gapminder-FiveYearData.csv
_episodes_rmd/02-project-intro.Rmd:174:> > wc -l data/gapminder-FiveYearData.csv
_episodes_rmd/02-project-intro.Rmd:178:> > head data/gapminder-FiveYearData.csv
_episodes_rmd/05-data-structures-part2.Rmd:233:gapminder <- read.csv("data/gapminder-FiveYearData.csv")
_episodes_rmd/05-data-structures-part2.Rmd:244:> download.file("https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/_episodes_rmd/data/gapminder-FiveYearData.csv", destfile = "data/gapminder-FiveYearData.csv")
_episodes_rmd/05-data-structures-part2.Rmd:245:> gapminder <- read.csv("data/gapminder-FiveYearData.csv")
_episodes_rmd/05-data-structures-part2.Rmd:250:> gapminder <- read.csv("https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/_episodes_rmd/data/gapminder-FiveYearData.csv")
_episodes_rmd/05-data-structures-part2.Rmd:365:> > download.file("https://raw.githubusercontent.com/swcarpentry/r-novice-gapminder/gh-pages/_episodes_rmd/data/gapminder-FiveYearData.csv", destfile = "data/gapminder-FiveYearData.csv")
_episodes_rmd/05-data-structures-part2.Rmd:366:> > gapminder <- read.csv(file = "data/gapminder-FiveYearData.csv")
_episodes_rmd/06-data-subsetting.Rmd:24:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/06-data-subsetting.Rmd:376:> gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/07-control-flow.Rmd:21:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/07-control-flow.Rmd:334:> > gapminder <- read.csv("data/gapminder-FiveYearData.csv")
_episodes_rmd/08-plot-ggplot2.Rmd:23:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header = TRUE)
_episodes_rmd/09-vectorization.Rmd:18:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/10-functions.Rmd:26:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/11-writing-data.Rmd:20:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/12-plyr.Rmd:18:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/13-dplyr.Rmd:24:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
_episodes_rmd/14-tidyr.Rmd:20:gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE, stringsAsFactors = FALSE)
```

**Comment by naupaka** [comment]
I will fix this now and submit/merge a PR

**Comment by naupaka** [comment]
Fixed by #424
